packages/slate/test/interfaces/Editor/unhangRange/text-hanging.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx jsx */
+import { Editor } from 'slate'
+import { jsx } from '../../..'
+
+export const input = (
+  <editor>
+    <block>
+      <text>
+        before
+        <anchor />
+      </text>
+      <text>selected</text>
```

Comment:
Also in #3683 it was guessed that `unhangRange` should also unhang the starting point of the selection (including Text nodes): https://github.com/ianstormtaylor/slate/issues/3683#issuecomment-634581973

I wonder if this does make sense with the original intention of this method. Does it?

Reply:
This would solve at least this issue I think: https://github.com/ianstormtaylor/slate/issues/4793